src/app/services/job/mock-job.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Build } from '../../models/Build';
+import { Job } from '../../models/Job';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/observable/of';
+
+import { BuildService } from '../build/build.service';
+import { JobService } from './job.service';
+import { Builds } from '../../mock/builds';
+import { Jobs } from '../../mock/jobs';
+import { filter } from 'rxjs/operators';
+
+export class MockJobService implements JobService {
+  getJobs(buildId: string): Observable<Job[]> {
+    const filteredList = Jobs.filter(
+     (job: Job) => job.build_id === buildId
```

Comment:
you shouldn't need to filter the list by build, since `getJobs()` takes `buildId` as a parameter, that returned jobs should be associated with build in question. And as you've discovered there's no `build_id` property on Job: https://github.com/Azure/brigade/blob/8d6c2dc96a1526f8e2b581f878c250abbb8aeb6f/pkg/brigade/job.go#L34